setup/native-script.rb

```diff
@@ -43,40 +43,18 @@
 puts "Installing the Java SE Development Kit... This might take some time, please, be patient. (You might need to provide your password.)"
 system('brew cask install java')
 system('echo "export JAVA_HOME=$(/usr/libexec/java_home)" >> ~/.bash_profile')
-system('echo "export ANDROID_HOME=/usr/local/opt/android-sdk" >> ~/.bash_profile')
-
-puts "Installing node.js 0.12"
-system('brew install homebrew/versions/node012')
-
-puts "Creating Homebrew formula for NativeScript."
-File.open("/usr/local/Library/Formula/native-script.rb", "w:utf-8") do |f|
-  f.write DATA.read
-end
 
-puts "Installing NativeScript formula... This might take some time, please, be patient."
-system('brew install native-script')
-
-__END__
-
-class NativeScript < Formula
-  desc "NativeScript"
-  homepage "https://www.nativescript.org"
-  version "1.3.0"
-  url "https://raw.githubusercontent.com/NativeScript/nativescript-cli/brew/setup/empty.tar.gz"
-  sha256 "813e1b809c094d29255191c14892a32a498e2ca298abbf5ce5cb4081faa4e88f"
+puts "Installing Android SDK"
+system('brew install android-sdk')
+system('echo "export ANDROID_HOME=/usr/local/opt/android-sdk" >> ~/.bash_profile')
```

Comment:
.profile might be the better option, for [it gets considered by both zsh and bash](https://en.wikipedia.org/wiki/Unix_shell). It will still not work in fancier shells like fish, but I suggest that bash and zsh are a good coverage.